docs/TutorEnvironmentManager.rst

```diff
@@ -0,0 +1,126 @@
+Tutor Environment Manager
+#########################
+
+Tutor environment manager allows you to create one project and set one version for it.
+
+You will be able to continue using the `tutor version manager`_ commands in your project:
+
+- tvm list
+- tvm plugins list
```

Comment:
It seems to me that this information should be a note and not have such a high priority.

```suggestion
**Note:** You can continue using the `tutor version manager`_ commands in your project. Example: `tvm list`, `tvm plugins list`.
```